metacat-common/src/main/java/com/netflix/metacat/common/dto/BusinessMetadataDto.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.netflix.metacat.common.QualifiedName;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+/**
+ * Business Metadata Dto.
+ * Business metadata should be at database/table level.
+ * @author zhenl
+ * @since 1.2.0
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class BusinessMetadataDto extends BaseDto implements HasBusinessMetadata {
+    private static final long serialVersionUID = -1234707236185689774L;
+    private QualifiedName name;
+    // Marked as transient because we serialize it manually, however as a JsonProperty because Jackson does serialize it
+    @ApiModelProperty(value = "businessmetadata")
+    @JsonProperty
+    private transient ObjectNode businessMetadata;
+
+    private void readObject(final ObjectInputStream in) throws IOException, ClassNotFoundException {
```

Comment:
Why are these methods needed? Jackson should be able to bind to `ObjectNode` by default.